src/components/generic-elements/call-to-action-button/call-to-action-button.test.js

```diff
@@ -0,0 +1,74 @@
+import { shallow, mount } from "enzyme";
+import CallToActionButton from "./call-to-action-button";
+import Modal from 'react-modal';
+import { AuthenticatedHTTPClient } from "../../../utils/http-clients";
+
+describe("call-to-action-button", () => {
+  test("it calls window.location.href onClick", done => {
+    AuthenticatedHTTPClient.get = jest.fn();
+    AuthenticatedHTTPClient.get.mockImplementationOnce(() => ({
+      status: 200,
+      data: "http://test.example.com/api/v1/presigned/url?key=123"
+    }));
+
+    const url = "http://dummy.com";
+    Object.defineProperty(window, "location", {
+      value: {
+        href: url
+      },
+      writable: true
+    });
+
+    let subject = shallow(
+      <CallToActionButton url={"abc"} format="csv" filename="dataset.csv" />
+    );
```

Comment:
this is a small thing, but I would like to see us use the pattern of the other tests where we have a function with overrides for default params. IMO, it makes it easier to see what the test is actually testing

Reply:
You got it Jobin!

Reply:
😎